exonum/src/storage/indexes_metadata.rs

```diff
@@ -154,8 +161,35 @@ pub fn assert_index_type(name: &str, index_type: IndexType, is_family: bool, vie
     }
 }
 
+#[derive(Debug, Clone)]
+pub(crate) enum StorageVersion {
+    Supported(VersionValue),
+    Unsupported(VersionValue),
+    Unspecified,
```

Comment:
I propose to remove `Unspecified` enum variant, because it is used only once. I propose to replace it with panic.